README.md

```diff
@@ -62,10 +61,9 @@ Chromium, FireFox, and IE:
     no ability to JIT.
   * **Heterogeneous Development Environments.** This is slowly evening out, but
     all CE devices run on custom hardware, often with proprietary methods of
-    building, packaging, deploying, and running programs. Almost all CE devices
-    have ARM processors instead of the more familiar x86. Sometimes the
-    toolchain doesn't support contemporary C++11/14 features. Sometimes the OS
-    isn't POSIX, or it tries to be, but it is only partially implemented.
+    building, packaging, deploying, and running programs. Sometimes the
+    toolchain doesn't support contemporary C++17 features. Sometimes the OS
+    does not support POSIX, or it is only partially implemented.
     Sometimes the program entry point is in another language or architecture
     that requires a "trampoline" over to native binary code.
   * **No navigation.** The point of a Single-Page Application is that you don't
```

Comment:
@andrewsavage1 There's a section below about "No scrolling" which really confuses me. What is it trying to say ?

Reply:
I'd guess it wasn't updated since hybrid nav/my work at all, I know I didn't make any changes to it (little weird that we list capabilities in this doc in the first place but oh well). Let's remove that bullet

Reply:
Removed the whole thing